Core/src/org/sleuthkit/autopsy/centralrepository/datamodel/PersonaAccount.java

```diff
@@ -275,6 +275,9 @@ public void process(ResultSet rs) throws CentralRepoException, SQLException {
      *                              persona_account.
      */
     public static Collection<PersonaAccount> getPersonaAccountsForAccount(long accountId) throws CentralRepoException {
+        String queryClause = PERSONA_ACCOUNTS_QUERY_CALUSE
+                + " WHERE persona_accounts.account_id = " + accountId
+                + "AND p.status_id != " + Persona.PersonaStatus.DELETED.getStatusId();
```

Comment:
There's a re-declaration error here, that got fixed on the parent branch. That needs to get merged in here.